### DISCUSSION TITLE
* (asterisk) parameter/argument in documentation

### DISCUSSION THREAD

**Discussion by gwynge** [micropython/micropython#9596] [General]
In several areas of documentation such as here [https://docs.micropython.org/en/latest/library/machine.Pin.html#machine-pin](url) there is a parameter in a function signature of *. This doesn't seem to be defined and is absent from the list of parameters.

For example: class machine.Pin(id, mode=- 1, pull=- 1, *, value=None, drive=0, alt=- 1)_

What is it?

**Comment by dlech** [comment]
It means that everything after the `*` is a keyword-only argument.  

More info: 
- https://stackoverflow.com/questions/14301967/bare-asterisk-in-function-arguments
- https://peps.python.org/pep-3102/

**Reply by gwynge** [reply]
Aha, slightly bonkers but, yup I get the idea! Thanks :o)
